06-reproj.Rmd

```diff
@@ -14,21 +14,16 @@ library(spDataLarge)
 
 ## Introduction
 
-<!-- A vital type of geometry transformation is *reprojecting* from one coordinate reference system (CRS) to another. -->
-<!-- Because of the importance of reprojection, introduced in Chapter \@ref(spatial-class), and the fact that it applies to raster and vector geometries alike, it is the topic of the first section in this chapter. -->
-
 Section \@ref(crs-intro) introduced coordinate reference systems (CRSs) and demonstrated their importance.
 This chapter goes further.
 It highlights issues that can arise when using inappropriate CRSs and how to *transform* data from one CRS to another.
 
-As illustrated in Figure \@ref(fig:vectorplots), there are two types of CRS: *geographic* ('lon/lat', with units in degrees longitude and latitude) and *projected* (typically with units of meters from a datum).
```

Comment:
I don't think this is an improvement. "There are two types of apple" is surely better than "there are two types of apples". Unless I'm missing something, the same should apply to types of Coordinate Reference System (CRS).

Reply:
@Nowosad looking into it - I take that back. I think it is an improvement so keeping it.